src/DDTrace/Integrations/CakePHP/V2/CakePHPIntegrationLoader.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace DDTrace\Integrations\CakePHP\V2;
+
+use CakeEvent;
+use CakeRequest;
+use DDTrace\GlobalTracer;
+use DDTrace\Integrations\CakePHP\CakePHPIntegration;
+use DDTrace\Integrations\Integration;
+use DDTrace\Contracts\Span;
+use DDTrace\Tag;
+use DDTrace\Type;
+use Router;
+
+class CakePHPIntegrationLoader
+{
+    /**
+     * @var Span
+     */
+    public $rootSpan;
+
+    public function load(CakePHPIntegration $integration)
+    {
+        // Very strange workaround to get the integration to load in tests
+        // We need to find this bug in the CLI SAPI's built-in web server
+        // Once the bug is fixed we can remove this line block
+        if ('true' === getenv('DD_TEST_INTEGRATION')) {
+            echo ' ';
+        }
+        $this->rootSpan = GlobalTracer::get()->getRootScope()->getSpan();
+        // Overwrite the default web integration
+        $this->rootSpan->setIntegration($integration);
+        $this->rootSpan->setTraceAnalyticsCandidate();
+        $this->rootSpan->overwriteOperationName('cakephp.request');
+        $this->rootSpan->setTag(Tag::SERVICE_NAME, CakePHPIntegration::getAppName());
+
+        $self = $this;
```

Comment:
I know this is a pattern we do already use. But It looks to me like the readability might be improved if its renamed to `$integration` instead of `$self`- WDYT?

Reply:
Nice catch! `$integration` is already being used in that scope so I changed it to `$loader`. :)